jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java

```diff
@@ -157,7 +157,7 @@ public void generateQuickStartWebXml(WebAppContext context, OutputStream stream)
         webappAttr.put("xmlns", "http://xmlns.jcp.org/xml/ns/javaee");
         webappAttr.put("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance");
         webappAttr.put("xsi:schemaLocation", "http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_" + (major + "_" + minor) + ".xsd");
-        webappAttr.put("metadata-complete", "true");
+        webappAttr.put("metadata-complete", "false");
```

Comment:
Surely this should not be hard coded to false, but take the value from the main web.xml
Of course there will be issues if some fragments are complete and others are not!!!

Reply:
Ah, good point. I'll fix.